components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
No need of committing the `.gitignore` file again since it is available at the repo root level.